examples/placegoose/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "placegoose",
+  "scripts": {
+    "dev": "wrangler dev src/index.ts",
+    "deploy": "wrangler deploy --minify src/index.ts",
+    "db:touch": "wrangler d1 execute placegoose-d1 --local --command='SELECT 1'",
+    "db:generate": "drizzle-kit generate",
+    "db:migrate": "wrangler d1 migrations apply placegoose-d1 --local",
+    "db:migrate:prod": "ENVIRONMENT=production drizzle-kit migrate",
+    "db:seed": "tsx _scripts/seed/index.ts",
+    "db:setup": "npm run db:touch && npm run db:generate && npm run db:migrate && npm run db:seed",
+    "fiberplane": "fpx",
```

Comment:
```suggestion
    "fiberplane": "pnpx @fiberplane/studio@latest",
```

Reply:
why not just install as a dep? on that note, i found both the instructions + the setup process for fpx a bit opaque.

i'd rather be told to install, configure, and run the package than be given an automagical script that will try to do stuff for you. e.g., i don't necessarily want to run the client as soon as i install it.